packages/config/config/devices/0x027a/zse43.json

```diff
@@ -0,0 +1,204 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZSE43",
+	"description": "Tilt Shock XS Sensor",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xe003"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 3,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Tilt On/Off",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Shock On/Off",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
```

Comment:
In the production firmware, this isn't an enable/disable parameter and instead has three options. Please update.

https://www.support.getzooz.com/kb/article/817-zse43-tilt-shock-xs-sensor-advanced-settings/